packages/xrpl/test/integration/requests/getAggregatePrice.test.ts

```diff
@@ -0,0 +1,78 @@
+import { stringToHex } from '@xrplf/isomorphic/dist/utils'
+import { assert } from 'chai'
+
+import { OracleSet } from '../../../src'
+import serverUrl from '../serverUrl'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import { testTransaction } from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('get_aggregate_price', function () {
+  let testContext: XrplIntegrationTestContext
+
+  beforeEach(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterEach(async () => teardownClient(testContext))
+
+  it(
+    'base',
+    async () => {
+      const tx: OracleSet = {
+        TransactionType: 'OracleSet',
+        Account: testContext.wallet.classicAddress,
+        OracleDocumentID: 1234,
+        LastUpdateTime: Math.floor(Date.now() / 1000),
```

Comment:
Should there be a helper function that does this math for users?

Reply:
Yeah that would be convenient. I can add something like `getUnixTime()`

Reply:
Done in https://github.com/XRPLF/xrpl.js/pull/2688/commits/41b9da15d902b2d62d8fd7328679036f17137126